build.zig

```diff
@@ -365,6 +365,18 @@ pub fn build(b: *std.Build) !void {
     test_deps.dependOn(test_fixture(b, "printenv-test", null, target, optimize, &flags));
     test_deps.dependOn(test_fixture(b, "streams-test", libuv, target, optimize, &flags));
 
+    // xxd - hex dump utility (vendored from Vim)
+    const xxd_exe = b.addExecutable(.{
```

Comment:
oh glad you noticed this. i think we forgot to update build.zig for `tee` ...

(not a blocker for this PR)